### PR TITLE
Multiple spell check fixes

### PR DESCRIPTION
**Pull request by timowestnosto**

### CODE REVIEW

File: implementing-nosto/implement-search/getting-started-with-nosto-search/implementing-category-pages.md [hunk 1]
```diff
@@ -33,9 +32,9 @@ init({
 ```
 {% endcode %}
 
-#### categoryQuery parameter as function&#x20;
+#### `categoryQuery` parameter as function&#x20;
```
Comment (timowestnosto) [inline implementing-nosto/implement-search/getting-started-with-nosto-search/implementing-category-pages.md]: Should we humanize instead?

File: implementing-nosto/implement-search/implement-search-using-code-editor/implementing-autocomplete.md [hunk 2]
```diff
@@ -38,9 +38,9 @@ init({
 ```
 {% endcode %}
 
-#### autocompleteQuery parameter as function&#x20;
+#### `autocompleteQuery` parameter as function&#x20;
```
Comment (timowestnosto) [inline implementing-nosto/implement-search/implement-search-using-code-editor/implementing-autocomplete.md]: Should we humanize instead?

File: implementing-nosto/implement-search/implement-search-using-code-editor/implementing-search-page.md [hunk 3]
```diff
@@ -30,9 +29,9 @@ init({
 })
 </code></pre>
 
-#### serpQuery parameter flavors&#x20;
+#### `serpQuery` parameter flavors&#x20;
```
Comment (timowestnosto) [inline implementing-nosto/implement-search/implement-search-using-code-editor/implementing-search-page.md]: Should we humanize instead?

File: implementing-nosto/implement-search/getting-started-with-nosto-search/implementing-category-pages.md [hunk 4]
```diff
@@ -33,9 +32,9 @@ init({
 ```
 {% endcode %}
 
-#### categoryQuery parameter as function&#x20;
+#### Category query parameter as function&#x20;
 
-In the example above, we supply autocomplete query parameters as an object. Additionally, categoryQuery parameters can also be supplied as a function. The function flavor can be used for building complex query parameters and provides access to other pre-defined configuration parameters. Section below shows an example of categoryQuery supplied as a function which provides the  product variationId by accessing the pre-defined categoryId() and categoryPath() methods from the default configuration.
+In the example above, we supply autocomplete query parameters as an object. Additionally, `categoryQuery` parameter can also be supplied as a function. The function flavor can be used for building complex query parameters and provides access to other pre-defined configuration parameters. Section below shows an example of `categoryQuery` supplied as a function which provides the  product `variationId` by accessing the pre-defined `categoryId` and `categoryPath` methods from the default configuration.
```
Comment (manikandan-ravikumar) [inline implementing-nosto/implement-search/getting-started-with-nosto-search/implementing-category-pages.md]: Is there an extra space between "provides the" and the "product"?

**Review by manikandan-ravikumar — APPROVED** [review]

File: implementing-nosto/implement-search/getting-started-with-nosto-search/implementing-category-pages.md (hunk 4 above)
Comment (timowestnosto) [reply implementing-nosto/implement-search/getting-started-with-nosto-search/implementing-category-pages.md]: fixed

**Pull request merged**